ansible/configs/config.json

```diff
@@ -0,0 +1,24 @@
+{
+    "Server": {
+      "PublicIp": "{{ public_ip.content }}",
+      "PublicPort": "{{ public_port }}",
+      "ListenIp": "{{ ansible_host }}",
+      "ListenPort": "{{ private_port }}"
+    },
+    "AntiCheat": {
+      "Enabled": true,
+      "BanIpFromGame": true
+    },
+    "Compatibility": {
+      "AllowFutureGameVersions": true,
+      "AllowVersionMixing": true
+    },
+    "Debug": {
+      "GameRecorderEnabled": true,
+      "GameRecorderPath": "{{ impostor_path }}/records/"
```

Comment:
We may need to move this later. I am attaching a [whole drive](https://github.com/rayprogramming/terraform-aws-impostor-server/pull/11/files#diff-6c27bcefa33af3496292d0448a95189bc1f75e38a9a6c9970e4490bef4c98a84R57) to hold records.

Reply:
Okay no problem, we can just pass that var into the provisioner when the playbook is run and use that instead